examples/window-app/common/window-app.zap

```diff
@@ -5109,7 +5109,7 @@
               "code": 0,
               "mfgCode": null,
               "source": "client",
-              "incoming": 0,
+              "incoming": 1,
```

Comment:
Do the actual example apps implement this?

Reply:
If it does not, then calling the method would return an error right? If so, maybe at least making the zap data definition spec-compilant is a start and certification would validate that methods actually work.

@AlexPygin any chance you can try it out with chip-tool and see what responses you get for newly added methods?

Reply:
> If it does not, then calling the method would return an error right?

But claiming you implement a thing when you don't is not spec-compliant.

(And it might not in fact return an error, depending.)

> maybe at least making the zap data definition spec-compilant is a start and certification would validate that methods actually work.

I feel like that's just sweeping the spec-compliance issues under the rug, and making them much harder to find (manual certification process vs the automated lint we are using here).

As in, we should make sure that there is at least one example app for the given .zap that actually implements the functionality, before we enable it in the .zap.